packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -581,10 +551,8 @@ contract Connext is
       });
     }
 
-    // Pay metatx relayer
-    // NOTE: if this is done *without* fast liquidity, router will be address(0) and the relayer
-    // will always be paid
-    _handleRelayerFees(_transferId, _args.routers, _args.feePercentage, _args.relayerSignature);
+    // Set the relayer for this transaction to allow for future claim
+    transferRelayer[_transferId] = msg.sender;
```

Comment:
Where is `transferRelayer` defined?

Reply:
It is defined in the same Connext contract [here](https://github.com/connext/nxtp/blob/relayer-fee-execute/packages/deployments/contracts/contracts/Connext.sol#L173
) it was introduced with 29bc25ed154e7827352c82479d8877253e830c5d